azurerm/config.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/Azure/azure-sdk-for-go/arm/containerservice"
```

Comment:
In that case, does the number of instances (`capacity`) within VMSS need to be set to computed?